charts/rasa-x/templates/rasa-x-deployment.yaml

```diff
@@ -73,8 +73,6 @@ spec:
           value: "0"
         - name: "SANIC_RESPONSE_TIMEOUT"
           value: "3600"
-        - name: "LOCAL_MODE"
```

Comment:
Removed this as it was a duplicate which was defined above.

Reply:
🚀